addons/flashlights/CfgVehicles.hpp

```diff
@@ -0,0 +1,45 @@
+class CfgVehicles {
+    class Man;
+    class CAManBase: Man {
+        class ACE_SelfActions {
+            //todo: add flashlight attach actions
+        }; 
+    };
+    
+    class NATO_Box_Base;
+    class EAST_Box_Base;
+    class IND_Box_Base;
+    class FIA_Box_Base_F;
+
```

Comment:
May I ask for red light flashlight? Little feature request I know but very
useful. Is this doable?
9 lip 2015 11:58 "bux578" <notifications@github.com> napisał(a):

> In addons/flashlights/CfgVehicles.hpp
> <https://github.com/acemod/ACE3/pull/1816#discussion_r34240108>:
>
> > @@ -0,0 +1,45 @@
> > +class CfgVehicles {
> > +    class Man;
> > +    class CAManBase: Man {
> > +        class ACE_SelfActions {
> > +            //todo: add flashlight attach actions
> > +        };
> > +    };
> > +
> > +    class NATO_Box_Base;
> > +    class EAST_Box_Base;
> > +    class IND_Box_Base;
> > +    class FIA_Box_Base_F;
> > +
>
> Oh, and add it the ACE misc ammo box please (if you haven't already).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/acemod/ACE3/pull/1816/files#r34240108>.
>